executors/kubeexec/georep.go

```diff
@@ -45,26 +45,22 @@ func (s *KubeExecutor) GeoReplicationCreate(host, volume string, geoRep *executo
 	if geoRep.SlaveSSHPort != 0 {
 		sshPort = fmt.Sprintf(" ssh-port %d ", geoRep.SlaveSSHPort)
 	}
-	cmd := fmt.Sprintf("gluster --mode=script volume geo-replication %s %s::%s create%s%s", volume, geoRep.SlaveHost, geoRep.SlaveVolume, sshPort, geoRep.ActionParams["option"])
-
-	if force, ok := geoRep.ActionParams["force"]; ok && force == "true" {
```

Comment:
Please think out to enable force flag only in case retry. To preserve as possible original behavior